rest_docs/rest-catalog-open-api.yaml

```diff
@@ -367,6 +367,90 @@ paths:
       - $ref: '#/components/parameters/namespace'
       - $ref: '#/components/parameters/table'
 
+    get:
+      tags:
+        - Catalog API
+      summary: Load a table from the catalog
+      operationId: loadTable
+      description:
+        Load a table from the catalog.
+
+        The response contains both configuration and table metadata. The configuration, if non-empty is used
+        as additional configuration for the table that overrides catalog configuration. For example, this
+        configuration may change the FileIO implemented used for the table.
+
+        The response also contains the table's full metadata.
+      responses:
+        200:
+          $ref: '#/components/responses/LoadTableResponse'
+        400:
+          $ref: '#/components/responses/BadRequestErrorResponse'
+        401:
```

Comment:
We might want to include `403` status code as well.  `401` pertains to authentication where `403` pertains to authorization.  It's worth distinguishing between the two.

Reply:
I agree with this. We should add 403 to all of the routes in a follow-up PR.

Reply:
+1 for adding 403 for all routes

Reply:
I can handle 403 in a follow up PR if we'd like.

Reply:
Yes, please do!

Reply:
https://github.com/apache/iceberg/issues/4001